src/components/map/Map.tsx

```diff
@@ -19,24 +20,30 @@ type Props = {
   onMaximizeButtonPress?: () => void;
   showsUserLocation?: boolean;
   style?: StyleProp<ViewStyle>;
-  zoom?: number;
 };
 
 const MARKER_ICON_SIZE = normalize(40);
 
 export const Map = ({
   geometryTourData,
   isMaximizeButtonVisible,
+  isPOI = false,
   locations,
   mapCenterPosition,
   mapStyle,
   onMapPress,
   onMarkerPress,
   onMaximizeButtonPress,
   style,
-  zoom = 0,
   ...otherProps
 }: Props) => {
+  const { globalSettings } = useContext(SettingsContext);
+  const { settings = {} } = globalSettings;
+  const { zoomLevelForMaps = {}, locationService = {} } = settings;
+
+  const showsUserLocation = otherProps.showsUserLocation ?? !!locationService;
+  const zoom = isPOI ? zoomLevelForMaps?.zoomForPOI : zoomLevelForMaps?.zoom;
```

Comment:
this way we have no fallback value anymore, right? how does the map render now for screens, that had zoom=0 or zoom=4 before, when there are no global settings set? i would suggest a destruction for these two variables here with certain default values.

Reply:
`zoomForPOI` and `zoom` could also be renamed to something that makes "a map with a lot markers" or "a map with just one marker" more clear, as i wrote in the other comment already.

Reply:
But I think we need to specify the zoom here, so why don't we change it like this?

`zoomValueForMultipleMarkerMap` und `zoomValueForOneMarkerMap`

Reply:
Yes, you are right. We're already talking about the zoom value for maps. No need to repeat it again.
How about this one?

`"zoomLevelForMaps": { "multipleMarkers": 4, "singleMarker": 1 }`

Reply:
great, I can't think of anything better

Reply:
I changed it.

https://github.com/smart-village-solutions/smart-village-app-app/commit/34ff12eaf22df69c02cde6db188b3c78f214a07c